massdriver-application-azure-vm-docker/init.tf

```diff
@@ -0,0 +1,17 @@
+locals {
+  environment_variables = [for env_key, env_value in module.application.envs : "-e ${env_key}=${env_value}"]
+
+  cloud_init_rendered = templatefile("${path.module}/templates/cloud-init.tmpl",
+    {
+      container_port        = var.port
+      container_image       = var.container.repository
+      container_tag         = var.container.tag
+      environment_variables = join(" ", local.environment_variables)
+    }
+  )
+}
+
+# debugging
+# output "cloud_init_rendered" {
+#   value = local.cloud_init_rendered
+# }
```

Comment:
Can these be yanked?

Reply:
I found it very useful debugging? Thoughts?